pyiron/atomistics/structure/analyse.py

```diff
@@ -130,4 +130,91 @@ def get_layers(self, distance_threshold=0.01, id_list=None, wrap_atoms=True, pla
             return np.asarray(layers).flatten()
         return np.vstack(layers).T
 
+    def pyscal_steinhardt_parameter(self, neighbor_method="cutoff", cutoff=0, n_clusters=2,
+                                            q=(4, 6), averaged=False, clustering=True):
+        """
+        Calculate Steinhardts parameters
+
+        Args:
+            job (job): pyiron job
+            neighbor_method (str) : can be ['cutoff', 'voronoi']
+            cutoff (float) : can be 0 for adaptive cutoff or any other value
+            n_clusters (int) : number of clusters for K means clustering
+            q (list) : can be from 2-12, the required q values to be calculated
+            averaged (bool) : If True, calculates the averaged versions of the parameter
+            clustering (bool) : If True, cluster based on the q values
+
+        Returns:
+            q (list) : calculated q parameters
+
+        """
+        from pyiron.atomistics.structure.pyscal import get_steinhardt_parameter_structure
```

Comment:
I guess you can import all functions at the top of the module, as `pyscal` is available on all three operation system it should not cause any issue. The only limitation is that it is currently not available as pip package.

Reply:
Apparently, this results in circular dependencies.

Reply:
I just had a brief look and couldn't see the issue.  Is there a detailed error?  I guess pyscal shouldn't have an import dependency on the atoms module.

Reply:
Well if I shift this upwards some tests fail (da86ce6) see with the error
`ImportError: cannot import name 'pyiron_to_ase' from partially initialized module 'pyiron.atomistics.structure.atoms' (most likely due to a circular import) (/home/runner/work/pyiron/pyiron/pyiron/atomistics/structure/atoms.py)`
In pyscal we import pyiron_to_ase from atoms, in atoms we import analyze, thus, we close the circle by importing pyscal in analyze.

Reply:
I pushed one commit that changes the import statement in the pyscal module.  This seems to fix the problems in my local testing.  There's an explanation [here](https://stackoverflow.com/questions/7336802/how-to-avoid-circular-imports-in-python/37126790#37126790), but I didn't fully understand.  It seems to be something weird about pythons import machinery.

If it doesn't work, feel free to
```
git reset --hard HEAD~
git push --force
```

Reply:
I also could just revert it - only the commit history would be messier... Speaking of commit history - is there a nice way to revert the revert?

Reply:
The easiest would be an interactive rebase + force push as [here at the bottom](https://github.com/eisenforschung/cm-onboarding/blob/master/Git.md).

Reply:
Well, I do not like force push very much. We could cleanup in the end this way if we really want to. Now I just revert the revert...

Reply:
Hopefully, this works now on every system! Another revert would really look extremely awkward...